src/App.tsx

```diff
@@ -1,3 +1,4 @@
+// tslint:disable:file-name-casing
```

Comment:
Is it not possible to just rename this file, too?

Reply:
There is some odd behavior when renaming this file.
The imports of this file insist that it is capitalized. Using lower case will result in an error when importing in the index.ts file. Though it resolves the TSLint error.

So I guess the decision is either between having the consistency with the file names or consistency with the imports. 

Since a wrong import is a more fatal error I chose to keep the import consistent; in case some future update will break everything.

Reply:
Hm, interesting... yeah in that case probably better to leave this way.